next.config.mjs

```diff
@@ -77,7 +85,7 @@ const nextConfig = {
 
   experimental: {
     instrumentationHook: true,
-    ppr: true,
+    // ppr: true,
```

Comment:
whats the plan with this feature (partial pre-rendering)? do we need to keep the commented config in the code?

Reply:
Good catch, I'll have to review why I commented it out.

Reply:
The PPR feature is only enabled in the latest canary versions and since we've switched back to their normal release branch the feature is no longer available.
![Screenshot 2024-04-22 at 1 13 05 PM](https://github.com/cds-snc/platform-forms-client/assets/25329319/f5c09b62-4c25-4a41-9f93-4905a5c0bb19)